Program.cs

```diff
@@ -1,25 +1,68 @@
 using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO.Compression;
+using System.IO;
 using System.Runtime.InteropServices.JavaScript;
+using System.Text.Json;
+using System.Text;
 using System.Threading.Tasks;
+using WasmBenchmarkResults;
 
 Console.WriteLine("Hello, Browser!");
-
-public partial class MyClass
+public partial class Program
 {
-    [JSExport]
-    internal static string Greeting()
+    readonly static string zipFileName = "index.zip";
+    readonly static string gitLogFile = "/git-log.txt";
+    readonly static string fileName = "index.json";
+
+    public static string getFlavor(string line)
     {
-        var text = $"Hello, World! Greetings from {GetHRef()}";
-        Console.WriteLine(text);
-        return text;
+        var words = line.Split("/");
+        StringBuilder stringBuilder = new();
+        for (var i = 2; i < words.Length - 1; i++)
+        {
+            stringBuilder.Append(words[i] + ".");
+        }
+
+        return stringBuilder.ToString().Remove(stringBuilder.Length - 1);
     }
 
-    [JSExport]
-    internal static Task<string> testMe()
+    internal static async Task<string> loadTests(string measurementsUrl)
     {
-        return Program.doSomth();
-    }  
+        var options = new JsonSerializerOptions { IncludeFields = true };
+        DataDownloader dataDownloader = new();
+        List<GraphPointData> list = new();
+        var bytes = await dataDownloader.downloadAsBytes(measurementsUrl + zipFileName);
+        var memoryStream = new MemoryStream(bytes);
```

Comment:
it is good practice to use `using` statement for classes implementing IDisposable interface (https://docs.microsoft.com/en-us/dotnet/standard/garbage-collection/using-objects)

you can extract a method for this part of code to load the items and use the using keywords on these like:
```
    private static async Task<List<Item>> LoadItems(string measurementsUrl, JsonSerializerOptions options)
    {
        DataDownloader dataDownloader = new();
        using var memoryStream = new MemoryStream(await dataDownloader.downloadAsBytes(measurementsUrl + zipFileName));
        using var archive = new ZipArchive(memoryStream);
        var entry = archive.GetEntry(fileName);
        using Stream readStream = entry.Open();
        using StreamReader streamReader = new StreamReader(readStream);
        return JsonSerializer.Deserialize<List<Item>>(streamReader.ReadToEnd(), options);
    }
```